tests/commands/dev/test_run_dev_app.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from subprocess import CalledProcessError
+import pytest
+
+from briefcase.exceptions import BriefcaseCommandError
+
+
+def raise_called_process_error(*args, **kwargs):
+    raise CalledProcessError(returncode=2, cmd="cmd")
```

Comment:
The extra callable isn't needed here; if the side effect is an instance of an exception, mock will cause it to be raised.